### PR TITLE
[CLI-2563] add example in confluent ksql cluster configureacls

### PR DESCRIPTION
**Pull request by MuweiHe**
Release Notes
-------------

Breaking Changes
- PLACEHOLDER

New Features
- PLACEHOLDER

Bug Fixes
- PLACEHOLDER

Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok

What
----

Adding an example per user's ask.

References
----------

Test & Review
-------------

Open Questions / Follow-ups
---------------------------

### CODE REVIEW

**Review by sgagniere — APPROVED** [review]

File: internal/ksql/command_cluster_configureacls.go [hunk 1]
```diff
@@ -24,6 +25,12 @@ func (c *ksqlCommand) newConfigureAclsCommand() *cobra.Command {
 		Args:              cobra.MinimumNArgs(1),
 		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
 		RunE:              c.configureACLs,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Configure ACLs for ksqlDB cluster lksqlc-12345 for topics topic_1 and topic_2:",
+				Code: "confluent ksql cluster configureacls lksqlc-12345 topic_1 topic_2",
```
Comment (brianstrauch) [inline internal/ksql/command_cluster_configureacls.go:31]:
```suggestion
				Text: `Configure ACLs for ksqlDB cluster "lksqlc-12345" for topics "topic-1" and "topic-2":`,
				Code: "confluent ksql cluster configure-acls lksqlc-12345 topic-1 topic-2",
```

Comment (MuweiHe) [reply internal/ksql/command_cluster_configureacls.go:31]: we're using my_topic in other examples so let's keep that

Comment (brianstrauch) [reply internal/ksql/command_cluster_configureacls.go]: We should definitely standardize this in a later PR, but sounds ok for now!

**Review by brianstrauch — APPROVED** [review]
Thank you!!

**Pull request merged**